models/9var/work/input.nml

```diff
@@ -144,9 +144,10 @@
 &preprocess_nml
     input_obs_def_mod_file = '../../../observations/forward_operators/DEFAULT_obs_def_mod.F90',
```

Comment:
Does anyone ever change 
input_obs_def
output_obs_def?

Reply:
not that i'm aware of, but if someone wanted to test a different version of the template they could use this facility.